### ISSUE TITLE
Swap Trick Not Working

### ISSUE THREAD

**Issue by hikari62** [slinga-homebrew/Save-Game-Extractor#10]
Hi, I've been trying to boot the disc on my Japanese model 1 Saturn for hours using swap trick and it just will not work. I'm not sure what I'm doing wrong, Has anyone successfully booted the SGEX ISO with swap trick? Or can anyone tell me what I'm doing wrong?I've watched every tutorial I could find to try to figure out what I'm doing wrong, but I just can't get it to work. 

What I've tried:
1) Turn on Saturn with SGEX disc, wait for the drive to spin down just as logo appears, swap to retail disc, wait for SEGA license screen to appear. At this point I have tried two different methods I've found, 2a and 2b.
2a) Hit reset as soon as license screen appears. Wait for saturn logo to appear and ask disc spins up, swap to SGEX disc. 
2b) Don't hit reset on license, instead swap back to SGEX disc after the access light flashes 3 times (indicating the security ring has been read I believe).

Neither of these methods works. With 2a, the Saturn hangs up on the license screen forever, just spinning the disc and doing nothing. With 2b sometimes it just goes to the home screen and I get the "disc unsuitable for system" error.  

I've found some forums indicating write speed might be an issue, so I tried 4x write speed on a new CDR, same issues. I've also found people saying maybe the smaller size of the disc changes the timing of the swap. I've tried adjusting my timing to no avail. I wrote the .iso to the CDR using Imgburn and I've tested the CDR on Yabause and it works fine there.

**Comment by slinga-homebrew** [comment]
Sorry to hear you are having so many issues. I just tried the swap trick on Save Game Extractor 0.95 on a US Saturn. It booted like a charm first try. 

1) Remove the top of your Saturn
2) Tape down the CD lid detector (it's in the back near the power). By taping it down the Saturn will always think the lid is closed. 
3) Insert the burned copy of SGEX. The one that you verified works in Yabause. 
4) Turn on your Saturn.
5) The disc will spin, a few seconds later you will hear the laser move slightly up. At that point swap it with an original Japanese Saturn CD. 
6) You will see the laser move to the outside of the disc. After a few seconds it will move back down to the center. Swap the original disc out for the SGEX CD. 

If all goes well it should boot. Takes like 20 seconds in total. If you are still having issues ask in #segaxtreme in Discord.

**Comment by hikari62** [comment]
It finally worked! Thank you! 

If anyone else reads this who is having trouble, following the above instructions about the laser and instructions about watching the lights I got on the discord (1. Put in the SGEX CD and turn on the saturn, wait for the access light to stop blinking, 2. Put in the original game, wait for the access light to blink 3 times, 3. Put in SGEX CD) did the trick. No resetting involved. The "reset at license screen version" of the swap trick I found online never did end up working for me, but that could be user error.
